sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/manager/MapOrPassthroughKeyMapper.java

```diff
@@ -24,7 +24,8 @@ public String toImpl(String value) {
         .orElseGet(
             () -> {
               LOG.warn(
-                  "No mapping to implementation found for key '{}' and value '{}'. Passing the value",
+                  "No mapping to implementation found for key '{}' and value '{}'. Passing the"
+                      + " value",
```

Comment:
Just saw, that this warning is mentioned in the docs. So I think, the warnings for using default values should also be mentioned. Thanks for adding this change to let me realize the log here. But I think, breaking the message is not needed:

```suggestion
                  "No mapping to implementation found for key '{}' and value '{}'. Passing the value",
```